lib/inertia_rails/controller.rb

```diff
@@ -17,5 +24,13 @@ def inertia_location(url)
       headers['X-Inertia-Location'] = url
       head :conflict
     end
+
+    def inertia_redirect_to(options = {}, response_options = {})
```

Comment:
I don't love the separate method name here because it feels a bit like a gotcha.. I can picture people wondering why errors aren't working and then having to say "oh yeah, you have to use inertia's special redirect_to". 

That being said I also share your unease about overwriting rail's `redirect_to` from a package..

No solution here, just thinking out loud.

Reply:
would you feel better about:

```ruby
redirect_to some_path, inertia_errors: { watch: 'out' }
```

or maybe

```ruby
redirect_to some_path, inertia: { errors: { watch: 'out' } }
```

the latter would give us flexibility to add more options in the future at the expense of being a bit more verbose.

i don't have a problem overriding `redirect_to` per se, just not with a generic key like `errors`. i want to scope this to inertia, but also in a way that will be intuitive for people

Reply:
I would actually, both of those seem really nice. No strong opinion between the two, but I probably lean towards the second since it feels like it adds an entire "inertia" section.

Reply:
cool, i will make that change